e2etest/zt_newe2e_oauth_device_test.go

```diff
@@ -0,0 +1,118 @@
+package e2etest
+
+import (
+	"flag"
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+	"github.com/Azure/azure-storage-azcopy/v10/testSuite/cmd"
+	"path"
+	"strings"
+	"time"
+)
+
+var runDeviceCodeTest = flag.Bool("device-code", false, "Whether or not to run device code tests. These must be run manually due to interactive nature.")
+
+var manualEnv AzCopyEnvironment
```

Comment:
Unused; code smell as documented in next comment

Reply:
this is accidental leftover from some of my debugging. will remove.